waspc/src/Wasp/Generator/Crud.hs

```diff
@@ -52,6 +53,9 @@ getCrudOperationJson crudOperationName crud idField =
 getCrudFilePath :: String -> String -> Path' (Rel r) File'
 getCrudFilePath crudName ext = fromJust (SP.parseRelFile (crudName ++ "." ++ ext))
 
+getCrudTypesImportPath :: String -> String
+getCrudTypesImportPath name = "wasp/server/crud/" ++ name
```

Comment:
Please make this consistent with the other stuff (location and implementation): https://github.com/wasp-lang/wasp/pull/1663/commits/59c0b6c72610708a9146d0de4437f3993d8fa05c#r1464996193